pexpect/pty_spawn.py

```diff
@@ -300,6 +300,10 @@ def preexec_wrapper():
         self.terminated = False
         self.closed = False
 
+    def spawnpty(self, args, **kwargs):
```

Comment:
I'm inclined to make this a private method for now - users should ~never call it, and I think it's only in fairly unusual circumstances that people should be overriding it.

@jquast, thoughts?